packages/components/Callout/src/Callout.types.ts

```diff
@@ -110,6 +110,56 @@ export interface ICalloutProps extends IViewProps, ICalloutTokens {
    */
   componentRef?: React.RefObject<IFocusable>;
 
+  /**
+   * Defines event redirection behaviors for pointer events relative to the Callout control.
```

Comment:
I wonder if some of this is better in a separate architecture file which can be linked to instead of in the code itself? I guess it's better that the note is as close to code as possible..

Reply:
probably -- will open a task to do this